sys/src/u-root.json

```diff
@@ -7,8 +7,8 @@
 			"GOARCH=amd64"
 		],
 		"Pre": [
-			"u-root -shellbang -initcmd= -o uroot.cpio plan9",
-			"lzma -f -k -3 uroot.cpio"
+			"u-root -shellbang -initcmd= -o /plan9_$ARCH/bin/uroot.cpio plan9",
```

Comment:
Is there a better place to put this besides `bin`? It's not an executable binary. Maybe just `/plan9_$ARCH/uroot.cpio`?

Reply:
fair point - changed